localstack/aws/protocol/service_router.py

```diff
@@ -354,7 +357,7 @@ def determine_aws_service_name(request: Request, services: ServiceCatalog = None
                 if len(services_per_prefix) == 1:
                     return services_per_prefix[0]
                 candidates.update(services_per_prefix)
-
+        print(f"{candidates=}")
```

Comment:
I have the commit ready to remove this, but the CI is green, so will push and merge straight after review is no change is necessary